pages/common/crictl.md

```diff
@@ -0,0 +1,36 @@
+# crictl
+
+> Command-line for CRI-compatible container runtimes.
+> More information: <https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md>.
+
+- List all kubernetes pods (Ready and NotReady):
+
+`crictl pods`
+
+- List all containers (Running and Exited):
+
+`crictl ps --all`
+
+- List all images:
+
+`crictl images`
+
+- Display information about one or more containers:
+
+`crictl inspect {{container_id}} {{container_id}}`
```

Comment:
```suggestion
- Print information about specific containers:

`crictl inspect {{container_id1 container_id2 ...}}`
```

Reply:
I think we have to decide what verbs we use: print, display or anything else. What about to stick to `print`/`list`? The second one use for list-like output and the first one for anything else.